### DISCUSSION TITLE
Tabs in create field flow

### DISCUSSION THREAD

**Discussion by azrikahar** [directus/directus#9654] [Feature Requests]
Tabs as a form of shortcut to interface groups without needing to scroll down:

![5AZ8NPADkY](https://user-images.githubusercontent.com/42867097/140943365-ed911915-383c-41ca-8b96-6b4bbfa8815b.gif)

Loosely relates to the suggestion here: https://github.com/directus/directus/discussions/9469#discussioncomment-1589302, but these are only tabs, not filterable so not sure how much better this is compared to current flow. Also not really mobile-responsive at the moment as it'll overflow to the right, but maybe we can just "hide" this since scrolling may still feel more natural on mobile? Just random thoughts here.

**Comment by rijkvanzanten** [comment]
I'm wondering if there's enough vertical scrolling right now to warrant the additional UI 🤔

**Reply by azrikahar** [reply]
That's true, hence my hesitation on the usefulness of this to general users. 

Side note: forgot to mention an edge case that prompted me thinking about this, which is my mouse scroll wheel haven't been working for quite a long time now and I've yet to get a new one 🤣

**Reply by benhaynes** [reply]
I like it! I think a big part of this depends on how extensions get sorted (in a category versus "other"). This does help us prepare for having more interfaces, and doesn't change the current flow too much (if you don't want to use it). Yes, we would have to disable these tabs on mobile, but then it's just the same as now, which is fine.

Perhaps we could also add a search bar to the header with autofocus, so as soon as you open the drawer you can type: `tag` and see that one interface... that would be nice too.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
